src/build_data_cmds.go

```diff
@@ -397,6 +397,7 @@ type BuildDataFetchCmd struct {
 	buildDataSingleCommitCommonOpts
 
 	DryRun bool `short:"n" long:"dry-run" description:"don't do anything, just show what would be done"`
+	Latest bool `short:"l" long:"latest" description: "fetch the latest successful build"`
```

Comment:
I don't think you're allowed to put a space between `description:` and `"fetch the latest successful build"`, that will probably not parse correctly.

[Go documentation](https://godoc.org/reflect#StructTag) states:

> By convention, tag strings are a concatenation of optionally space-separated key:"value" pairs.